src/Model/RestAPIMethod.cs

```diff
@@ -127,6 +130,11 @@ public RestAPIMethod(string requestContentType, IType returnType, bool isPagingN
         /// </summary>
         public AutoRestMethod AutoRestMethod { get; }
 
+        /// <summary>
+        /// Get whether or not this method is resumable.
```

Comment:
Can you note that resumable refers to resuming polling of an LRO?

Reply:
Done